.github/workflows/publish.yml

```
@@ -0,0 +1,62 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - "**"
+
+jobs:
+  lint:
+    if: github.ref == 'refs/heads/main' && github.actor == 'bot-anik'
+    uses: ./.github/workflows/lint.yml
+
+  build:
+    if: github.ref == 'refs/heads/main' && github.actor == 'bot-anik'
+    uses: ./.github/workflows/build.yml
+
+  test:
+    if: github.ref == 'refs/heads/main' && github.actor == 'bot-anik'
+    uses: ./.github/workflows/test.yml
+
+  perform-publish:
+    if: github.ref == 'refs/heads/main' && github.actor == 'bot-anik'
+    needs:
+      - lint
+      - build
+      - test
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.OPS_TOKEN }}
+
+      - name: Setup rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: 1.75
+          default: true
+          override: true
+
+      - name: Setup cmake
+        uses: jwlawson/actions-setup-cmake@v2
+        with:
+          cmake-version: "3.29.0"
+
+      - name: Cargo install smart-release
+        run: cargo install cargo-smart-release
+
+      - name: Publish crates to crates.io
+        env:
+          CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
+        run: |
+          cargo smart-release --execute --update-crates-index --no-push --no-changelog --no-changelog-github-release \
+          axone-rdf \
+          axone-wasm \
+          axone-objectarium-client \
+          axone-logic-bindings \
+          axone-cognitarium-client \
+          axone-objectarium \
+          axone-cognitarium \
+          axone-law-stone \
+          axone-dataverse
```

Another suggestion is to dynamically generate the list of crates to publish, avoiding forgetting future packages or smart contract creation. I don't know if the order is important, but if not, this should be good (I put in priority the packages list then contracts) 🙂.

```suggestion
          PACKAGES=$(find packages contracts -mindepth 1 -maxdepth 1 -type d -exec basename {} \; | tr '\n' ' ')
          echo $PACKAGES | xargs cargo smart-release --execute --update-crates-index --no-push --no-changelog --no-changelog-github-release
```